.github/workflows/upgrade_test_constraints.yml

```
@@ -13,16 +13,46 @@ on:
     paths:
       - '.github/workflows/upgrade_test_constraints.yml'
 
-concurrency:
-  group: test-${{ github.ref }}
-  cancel-in-progress: true
-
 jobs:
   upgrade:
     name: Upgrade & Open Pull Request
-    if: (github.event.issue.pull_request != '' && contains(github.event.comment.body, '@napari-bot update constraints')) || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule'
+    if: (github.event.issue.pull_request != '' && contains(github.event.comment.body, '@napari-bot update constraints')) || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event_name == 'pull_request'
     runs-on: ubuntu-latest
     steps:
+      - name: Add eye reaction
```

```suggestion
      - name: Add eyes reaction
```

Potentially I'd add a comment here about why do this...

---

(seems cute but unnecessary...)

---

Information that workflow start working. That command was properly triggered.